exchangelib/services.py

```diff
@@ -507,7 +507,15 @@ def _get_elements_in_response(self, response):
                         yield c
             except ErrorInvalidIdMalformedEwsLegacyIdFormat as e:
                 msg_text = get_xml_attr(msg, '{%s}MessageText' % MNS)
-                log.error('Caught ErrorInvalidIdMalformedEwsLegacyIdFormat error', error=str(e), exc_info=True, response_msg=msg_text)
+                log.error(
+                    'Caught ErrorInvalidIdMalformedEwsLegacyIdFormat'
+                    '\nerror: %s'
+                    '\nmsg_response: %s',
```

Comment:
Do we need the `\n`? We don't usually put those in error messages do we?

Reply:
its not necessary but could be done, I am doing this because `msg_response` might be a bit long so adding line breaker might make it easier to read

Reply:
I'm slightly worried the logs won't like it, but we can try leaving it in for now.